conf/logs.json

```diff
@@ -308,5 +308,526 @@
+      "referenced_tree": "string",
+      "smb1_offered_dialects": "string",
+      "smb2_offered_dialects": "string"
+    }
+  },
+    "bro:smb_files.log": {
+    "parser": "json",
+    "schema": {
+      "ts": "string",
+      "uid": "string",
+      "id": "string",
+      "fuid": "string",
+      "action": "string",
+      "path": "string",
+      "name": "string",
+      "size": "string",
+      "prev_name": "string",
+      "times": "string",
+      "fid": "string",
+      "uuid": "string"
+    }
+  },
+    "bro:smb_mapping.log": {
+    "parser": "json",
+    "schema": {
+      "ts": "string",
+      "uid": "string",
+      "id": "string",
+      "path": "string",
+      "service": "string",
+      "native_file_system": "string",
+      "share_type": "string"
+    }
+  },
+    "bro:smtp.log": {
+    "parser": "json",
+    "schema": {
+      "ts": "string",
+      "uid": "string",
+      "id": "string",
+      "trans_depth": "string",
+      "helo": "string",
+      "mailfrom": "string",
+      "rcptto": "string",
+      "date": "string",
+      "from": "string",
+      "to": "string",
+      "cc": "string",
+      "reply_to": "string",
+      "msg_id": "string",
+      "in_reply_to": "string",
+      "subject": "string",
+      "x_originating_ip": "string",
+      "first_received": "string",
+      "second_received": "string",
+      "last_reply": "string",
+      "path": "string",
+      "user_agent": "string",
+      "tls": "boolean",
+      "process_received_from": "boolean",
+      "has_client_activity": "boolean",
+      "entity": "string",
+      "fuids": "string",
+      "is_webmail": "boolean"
+    }
+  },
+    "bro:snmp.log": {
+    "parser": "json",
+    "schema": {
+      "ts": "string",
+      "uid": "string",
+      "id": "string",
+      "duration": "string",
+      "version": "string",
+      "community": "string",
+      "get_requests": "string",
+      "get_bulk_requests": "string",
+      "get_responses": "string",
+      "set_requests": "string",
+      "display_string": "string",
+      "up_since": "string"
+    }
+  },
+    "bro:socks.log": {
+    "parser": "json",
+    "schema": {
+      "ts": "string",
+      "uid": "string",
+      "id": "string",
+      "version": "string",
+      "user": "string",
+      "password": "string",
+      "status": "string",
+      "request": "string",
+      "request_p": "string",
+      "bound": "string",
+      "bound_p": "string"
+    }
+  },
+    "bro:ssh.log": {
+    "parser": "json",
+    "schema": {
+      "ts": "string",
+      "uid": "string",
+      "id": "string",
+      "version": "string",
+      "auth_success": "boolean",
+      "auth_attempts": "string",
+      "direction": "string",
+      "client": "string",
+      "server": "string",
+      "cipher_alg": "string",
+      "mac_alg": "string",
+      "compression_alg": "string",
+      "kex_alg": "string",
+      "host_key_alg": "string",
+      "host_key": "string"
+    }
+  },
+    "bro:ssl.log": {
```

Comment:
@landonix  Same comment as before, all of the `boolean` types here need to be strings since Bro's bool type has values of `F` and `T` per: https://www.bro.org/sphinx/script-reference/types.html#type-bool

Please carefully check this for all other log types